Libraries/Components/View/ViewPropTypes.js

```diff
@@ -170,7 +201,7 @@ type GestureResponderEventProps = $ReadOnly<{|
    *
    * See http://facebook.github.io/react-native/docs/view.html#onresponderterminationrequest
    */
-  onResponderTerminationRequest?: ?Function,
+  onResponderTerminationRequest?: ?(event: PressEvent) => mixed,
```

Comment:
Can you look to see what native does? Does iOS and Android code provide the argument?

Reply:
I can't quite figure out whether or not it does , and I cannot seem to contrive an example where this event will fire so I can't do a good ol' `console.log(event)` to see at runtime if it's getting an event. (If someone who knows their way around it better can provide an example where this event will fire, that would be handy! Maybe we can automatically test these in the future)

However, because PanResponder has the argument on its callback, if you spread the panresponder you get from `PanResponder.create()` on to a View, there will be a flow error. So to get around this I may need to modify `PanResponder`, which I'm not sure if I should do or not.

Reply:
To avoid blocking this diff until we know absolutely everything, should we instead leave the ones we aren't sure of as `Function` and land the rest?

Reply:
I would have to remove `strict-local` to do that (there's flow errors otherwise). If you're okay with putting off having `strict-local` on this file (or, I do a `$FlowFixMe`) for now perhaps we could do that

Reply:
They will have all of the properties that are in `PressEvent` (because they are an implementation of W3C touch events), but the only uncertainty I had was whether or not the `TouchEvent` I made was redundant.